src/main/java/server/Server.java

```diff
@@ -1,21 +1,40 @@
 package main.java.server;
 
-import java.io.IOException;
-import java.io.OutputStream;
+
+import lombok.extern.java.Log;
+import main.java.message.Request;
+import main.java.message.Response;
+
+import java.io.*;
 import java.net.ServerSocket;
 import java.net.Socket;
 
-
+@Log
 public class Server {
 
     private static final int PORT = 8080;
 
     public static void main(String[] args) {
-        try (Socket socket  = new ServerSocket(PORT).accept()) {
-            OutputStream stream = socket.getOutputStream();
-            stream.write("helloWorld".getBytes());
+
+        connetClient();
+    }
+
+    private static void connetClient() {
+        try (Socket socket = new ServerSocket(PORT).accept()) {
+            // input
+            BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            Request request = Request.init(in);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "request" local variable. ([squid:S1481](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS1481))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=4&issues=AWQ3A4bAbXXTsx7OVQgs&open=AWQ3A4bAbXXTsx7OVQgs)

[ISSUE_KEY]: AWQ3A4bAbXXTsx7OVQgs